.github/workflows/autogpt-builder-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: AutoGPT Builder CI
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'rnd/autogpt_builder/**'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'rnd/autogpt_builder/**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+    - name: Install dependencies
+      run: |
+        cd rnd/autogpt_builder
+        npm install
+    - name: Run lint
+      run: |
+        cd rnd/autogpt_builder
```

Comment:
You can avoid the repeated `cd` like this:
https://github.com/Significant-Gravitas/AutoGPT/blob/6e1c9d44a4db87d508059eee180788519475637a/.github/workflows/autogpt-ci.yml#L21-24